apps/wing/src/commands/test/test.ts

```diff
@@ -29,11 +29,23 @@ const generateTestName = (path: string) => path.split(sep).slice(-2).join("/");
  * Options for the `test` command.
  */
 export interface TestOptions extends CompileOptions {
-  /** Whether to keep the build output. */
+  /**
+   * Whether the output artifacts should be kept or cleaned up after the test run.
+   */
   clean: boolean;
+  /**
+   * The name of the output file.
+   */
   outputFile?: string;
-  /** String representing a RegEx used for test filtering. */
+  /**
+   * String representing a regex pattern used to selectively filter which tests to run.
+   */
   testFilter?: string;
+  /**
+   * How many times failed tests should be retried.
+   * @default 3
```

Comment:
In the context of calling test(), the default is actually 0
```suggestion
   * @default 0
```

Reply:
You are right, if you call `test()` directly the default value would be indeed `0`. It's Commander that sets the value to `3` when calling the method through it.

Reply:
You are right, if you call `test()` directly the default value would be indeed `0`. It's Commander that sets the value to `3` or `x` when calling the method through it.

Reply:
Actually, there's no default value. If you call `test()` directly, the `retry` variable would be undefined. It's Commander that sets the value but that doesn't exist without it.

Reply:
It's the reason why I manually set the value to `0` here: https://github.com/winglang/wing/pull/4666/files#diff-cc35dcb1d186df3226515f587ca678d51e6c8e1af1ef38684e312d30bbcab5daR286